app/uk/gov/hmrc/tradergoodsprofiles/models/responses/GetRecordResponse.scala

```diff
@@ -79,13 +78,20 @@ object GetRecordResponse {
         "nirmsNumber"              -> Json.toJson(o.nirmsNumber),
         "niphlNumber"              -> Json.toJson(o.niphlNumber),
         "locked"                   -> Json.toJson(o.locked),
-        "srcSystemName"            -> Json.toJson(o.srcSystemName),
         "createdDateTime"          -> Json.toJson(o.createdDateTime),
         "updatedDateTime"          -> Json.toJson(o.updatedDateTime)
       )
-      JsObject(fields)
+      removeNulls(JsObject(fields))
     }
 
+    private def removeNulls(jsObject: JsObject): JsObject =
```

Comment:
This method exists twice in this microservice, can it be placed somewhere common